src/lib/io/sol-iio.c

```diff
@@ -88,6 +89,15 @@ struct resolve_absolute_path_data {
     int id;
 };
 
+struct open_list {
```

Comment:
This struct is not actually the list, but an entry/item of the list. Maybe a more explicit name, to avoid confusion?